src/app/psat/explore-opportunities/explore-opportunities-results/explore-opportunities-results.component.html

```diff
@@ -16,8 +16,24 @@
     </div>
     <div class="d-flex align-items-center pl-1 pr-1" [ngClass]="{'col-4': !inSetup, 'col-6': inSetup}">
       <span class="table-text"> &mdash; &mdash;</span>
+    </div> 
+    <div class="col-4 d-lg-none d-flex align-items-center pl-1 pr-1" *ngIf="!inSetup">
+      <span *ngIf="(percentSavings && psat.modifications[exploreModIndex].psat.valid.isValid) && (percentSavings >= 0)" class="table-text">
+        {{percentSavings | number:'1.0-1'}}%
+      </span>
+      <span *ngIf="(percentSavings && psat.modifications[exploreModIndex].psat.valid.isValid) && (0 > percentSavings)" class="table-text">
+        0%
+      </span>
+      <div class="alert-danger small" *ngIf="!psat.modifications[exploreModIndex].psat.valid.isValid">
```

Comment:
In the future,  it would be better to pass the opportunityPsatValid, selectedModification object, etc.. down into this component (or set it in this component), and check object exists, rather than indexing into the modification valid object. Nicer to read and one less point of failure to debug.